### PR TITLE
Implemented moveItemInFormArray() function for cdk drag and drop, equivalent to moveItemInArray()

### PR DESCRIPTION
**Pull request by ngfelixl**
In this PR I've added a `moveItemInFormArray()` function in the **drag-utils.ts** for cdk drag and drop, since I haven't found any issues related to formArrays and some questions on stackoverflow regarding this topic.

### CODE REVIEW

File: src/cdk/drag-drop/drag-utils.ts [hunk 1]
```diff
@@ -5,6 +5,7 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+import { FormArray } from '@angular/forms';
```
[the remark this answers is not in the document]
Comment (crisbeto) [reply src/cdk/drag-drop/drag-utils.ts]: It might be worth asking over on the https://github.com/angular/angular repo about it.

**Comment by ngfelixl** [comment]
Made a feature request [#27171](https://github.com/angular/angular/issues/27171) and PR [#27222](https://github.com/angular/angular/pull/27222) in @angular/angular.

**Comment by gopalkrish23** [comment]
@ngfelixl  please give a working stackblitz example for this moveItemInFormArray() function. I need this feature in my project. Thanks :)

**Comment by ngfelixl** [comment]
@gopalkrish23 You can have a look at this [implementation](https://github.com/ngfelixl/documentations/blob/master/projects/documentations/src/lib/docu-editor/components/documentation-form.component.ts#L67) which works with reactive forms and cdk-drag-drop.

**Pull request closed without merging**